Lib/pathlib/types.py

```diff
@@ -397,7 +420,7 @@ def write_text(self, data, encoding=None, errors=None, newline=None):
         with magic_open(self, mode='w', encoding=encoding, errors=errors, newline=newline) as f:
             return f.write(data)
 
-    def _copy_from(self, source, follow_symlinks=True):
+    def _copy_from(self, source: _ReadablePath, follow_symlinks: bool = True) -> None:
```

Comment:
@barneygale I noticed `_copy_from` is currently less strict with additional kwargs compared to copy() and copy_into(). Shouldn't these be keyword only args?

```suggestion
    def _copy_from(self, source: _ReadablePath, *, follow_symlinks: bool = True) -> None:
```

Reply:
Good spot, yes you're right!